sdcm/cluster_k8s/__init__.py

```diff
@@ -2313,9 +2316,8 @@ def decommission(self, node):
                                  namespace=self.namespace,
                                  timeout=node.pod_terminate_timeout * 60 + 10)
         self.terminate_node(node, scylla_shards=scylla_shards)
-        # TBD: Enable rack deletion after https://github.com/scylladb/scylla-operator/issues/287 is resolved
-        # if current_members - 1 == 0:
-        #     self.delete_rack(rack)
+        if current_members - 1 == 0:
+            self._delete_k8s_rack(rack)
```

Comment:
Why not just `if current_members == 1:` ?

Reply:
Done